pkg/cli/admin/upgrade/upgrade.go

```
@@ -245,8 +245,11 @@ func (o *Options) Run() error {
 		return nil
 
 	case o.ToMultiArch:
+		if cv.Status.Desired.Architecture == configv1.ClusterVersionArchitectureMulti {
+			return errors.New("Cluster is already updating to multi-architecture")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)